ddtrace/profiling/profiler.py

```diff
@@ -129,6 +120,33 @@ def version(self):
     def tracer(self):
         return self._profiler.tracer
 
+    @property
+    def url(self):
+        return self._profiler.url
+
+
+def _get_default_url(api_key):
+    """Get default profiler exporter URL.
+
+    If an API key is not specified, the URL will default to the agent location configured via the environment variables.
+
+    If an API key is specified, the profiler goes into agentless mode and uses `DD_SITE` to upload directly to Datadog
+    backend.
+    """
+    # Default URL changes if an API_KEY is provided in the env
```

Comment:
Should we prioritize the url first? I think the UI now nags you to stop using the agentless configuration for profiling. This is slightly annoying if you run the agent in the same environment as the trace client, so DD_API_KEY is defined and used by default by the profiler.

Reply:
The URL is always used if specified (see L180), this is just the default in the user did not specify anything. In that case, we keep doing what we did so far.

Reply:
:+1: